shell/packages/sandstorm-ui-applist/applist-client.js

```diff
@@ -137,8 +137,10 @@ Template.sandstormAppList.helpers({
       return [];
     }
   },
-  origin: function() {
-    return document.location.protocol + "//" + document.location.host;
+  appMarketUrl: function() {
+    var appMarket = Settings.findOne({_id: "appMarketUrl"});
+    // If the Settings subscription hasn't yet propagated client-side, default to app.sandstorm.io
```

Comment:
This could be bad for some future users who really insist on using a specific app market. I'd remove the default. We probably shouldn't display anything until the settings collection is loaded anyhow...